src/cci/cas_cci.c

```diff
@@ -132,6 +132,9 @@ int wsa_initialize ();
 #define CON_HANDLE_ID_FACTOR            1000000
 #define CON_ID(a) ((a) / CON_HANDLE_ID_FACTOR)
 #define REQ_ID(a) ((a) % CON_HANDLE_ID_FACTOR)
+
+#define MAX_CON_HANDLE                  2048
```

Comment:
@airnet73 suggested that this can occur the problem. I agree this idea and I will solve by another code.